src/app/tasks/tasks-list/task-item/directives/drag.directive.ts

```diff
@@ -0,0 +1,90 @@
+import { Directive, ElementRef, inject, Renderer2, Input, OnInit, HostListener } from '@angular/core';
+
+export interface ElemActions {
+  onDrag?: () => void;
+  onClick?: () => void;
+}
+
+@Directive({
+  selector: '[appDrag]',
+  standalone: true
+})
+export class DragDirective implements OnInit {
```

Comment:
It seems like you some bugs in this behavior.

Please describe what you meant to do and what the correct behavior is

Reply:
it adds drag n' drop feature to an element.
in this case, when the task is dragged right it is removed, when clicked it goes to details.